src/types/ActivityTypes.tsx

```diff
@@ -8,6 +8,7 @@ export type QuizzAnswersType = {
   card: string,
   answerList:
```

Comment:
Ici tu pourrais deja creer un type `QcmAnswerType` de sorte a ce que QuizzAnswersList soit egal a : `export type QuizzAnswersType = { _id: string, card: string, answerList: QcmAnswerType[] };` non ?

Reply:
oui on peut mais en étant conscient que potentiellement ça sautera (que ça soit un type qcmanswerType ou le format actuel) quand on rajoutera le cas order the sequence (cf strat technique de order the sequence)